lib/Service/RetentionService.php

```diff
@@ -130,7 +130,7 @@ public function runCron(): void {
 	}
 
 	public function executeRetentionPolicy(IUser $user): ?bool {
-		$this->logger->warning($user->getUID());
+		$this->logger->debug($user->getUID());
```

Comment:
```suggestion
```

This was from debugging, I don't think we need it as any other path will log something instead.